_component_design/tables-controls.md

```diff
@@ -0,0 +1,281 @@
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+<div id="within" class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+## Table controls - within
+{: .hxSectionTitle}
+The following controls may be found inside a table:
+- Select Icon
+- Action Cog Button
+- Sort table actions
+- Expand Table Row
+- Retract / Truncate table row
+ 
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Definition of within table actions and spacing specifications. NOT TO SCALE"] [class:"image bg-light border"] %}          
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-controls-within.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div id="within" class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Actions cog
+{: .hxSubSectionTitle}
+
+The actions cog allows users to quickly initiate a number of create and edit workflows on an item in a table without drilling into an item’s detail page. These available actions are hidden behind the cog icon, allowing a user to interact with the cog to execute these actions. In obscuring these options behind a user click, the actions cog does not overload the users view of the table data.
+
+To ensure proper usage following these guidelines:
+
+- Only use a cog if you have three or more actions that may be performed on a table row.
+- Keep the actions list to seven or fewer actions.
+- Group like actions in the dropdown and avoid grouping unlike options.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Actions cog triggered IxD. NOT TO SCALE"] [class:"image bg-light border"] %}          
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-actions-ixd.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div id="within" class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Variations and icon spacing
+{: .hxSubSectionTitle}
+
+
+To ensure proper usage following these guidelines:
+
+- If a checkbox is present it **will always** be the first option
+- If the actions cog is present and a checkbox is not present it **will be the first option**
+- Icons should be 16px wide with 20px spacing on both sides.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Variations best practices and spacing. NOT TO SCALE"] [class:"image bg-light border"] %}          
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-actions-spacing.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div id="within" class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Bulk selections
```

Comment:
Are we aiming to use singular for headings? Not sure what the current consensus is.

Reply:
yeah, we seem to be loosely standardizing on this. changed.